### DISCUSSION TITLE
Turn overlay on/off with key command

### DISCUSSION THREAD

**Discussion by movieboy** [scribeocr/scribeocr#61] [Ideas]
The overlay is certainly great for quickly spotting errors in the OCR, but sometimes, when making corrections, it would be nice to be able to quickly turn the overlay off to confirm the accuracy and make edits.

**Comment by Balearica** [comment]
Thanks for the suggestion.  While the overlay opacity can be adjusted already using `View` -> `Overlay Opacity`, I agree that it would be nice to have a quicker way to briefly disable the overlay text.  I will add a shortcut for this.

**Reply by Balearica** [reply]
@movieboy I added a shortcut for this.  You can now temporarily hide all overlay contents by holding down `Ctrl + Space`.

**Comment by movieboy** [comment]
This is amazing!  Makes corrections so much faster!  Thanks!

**Reply by Balearica** [reply]
Glad this was useful, marking the thread as resolved.
